documentation/source/simulator_support.rst

```diff
@@ -256,6 +256,35 @@ Support is preliminary.
 Noteworthy is that despite GHDL being a VHDL simulator, it implements the VPI interface.
 
 
+.. _sim-ghdl-waveforms:
+
+Waveforms
+---------
+
+To get waveforms in VCD format, set the :make:var:`SIM_ARGS` option to ``--vcd=anyname.vcd``,
+for example in a Makefile:
+
+  .. code-block:: make
+
+    SIM_ARGS+=--vcd=anyname.vcd
+
+The option can be set on the command line, as shown in the following example.
+
+  .. code-block:: bash
+
+    make SIM=ghdl SIM_ARGS=--vcd=anyname.vcd
+
+    # or define the SIM_ARGS environment variable in bash
+
+    export SIM_ARGS=--vcd=anyname.vhd; make SIM=ghdl
+
+    # or in csh
+
+    setenv SIM_ARGS=--vcd=anyname.vhd; make SIM=ghdl
```

Comment:
Didn't we decide that we want to recommend none of these, but instead:
```suggestion
The option can be set on the command line, as shown in the following example.

.. code-block:: bash

    SIM_ARGS=--vcd=anyname.vhd make SIM=ghdl
```

Reply:
Frankly though, I don't think it's the job of `simulator_support.rst` to tell users how to set make variables - the page listing make variables should cover that.